api/migrations/0015_auto_20200426_1530.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.0.5 on 2020-04-26 12:30
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
We should have 1 migration per PR, I'll explain more tomorrow but basically:
- You unapply the new migrations
- Delete the new migrations
- Generate a new single migration file